tests/Feature/Modules/Core/CreditNoteCrudTest.php

```
@@ -0,0 +1,220 @@
+            'resource_permission' => 'global',
+        ])->save();
+
+        $this->currency = Currency::create([
+            'name'           => 'USD',
+            'symbol'         => '$',
+            'iso_numeric'    => 840,
+            'decimal_places' => 2,
+            'full_name'      => 'US Dollar',
+            'rounding'       => 0.00,
+            'active'         => true,
+        ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Core\Models\Currency' in method 'setUp'. (undefined)

(StaticAccess)